cmd/ocm/edit/ingress/cmd.go

```diff
@@ -55,6 +67,12 @@ const (
 	namespaceOwnershipPolicyFlag  = "namespace-ownership-policy"
 	clusterRoutesHostnameFlag     = "cluster-routes-hostname"
 	clusterRoutesTlsSecretRefFlag = "cluster-routes-tls-secret-ref"
+	componentRoutesFlag           = "component-routes"
+
+	amountOfParametersPerComponentRoute = 2
+	hostnameParameter                   = "hostname"
```

Comment:
nit: indenting seems off

Reply:
This one is just the separation from the parameters above which are all flags